interview-questions.md

```diff
@@ -8,42 +8,42 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is object-oriented programming? How is it different than functional programming?
 
-Your answer:
+Your answer: I am not sure what object-oriented programming is but I know that objects can be used when creating functions. Functional programming seems to be focused on the functions at hand. 
 
-Researched answer:
+Researched answer: Object-oriented programming is a lot more popular then functional programming. It has also been around longer. Object-oriented programming uses methods and objects as key coding elements. And Functional programming uses variables and functions.
 
 2. What is the difference between a Float and an Integer in Ruby?
 
-Your answer:
+Your answer: Float refers to a floating number, as far as knowing what that means I am going to research and let you know as soon as I find out. An integer in Ruby I know is a number with no strings around it because that would then make the item a string. 
 
-Researched answer:
+Researched answer: An integer is a number that has no decimal. And a floating number is a number that has a decimal point. For example: floating number--> 3.3 and integer-->3
 
 3. Ruby has an implicit return. What does that mean?
 
-Your answer:
+Your answer: I think that this is referring to the error where every function in Ruby needs to have a return in order for it to function properly. After this interview, I would love to send you what I find after researching more into this topic. 
 
-Researched answer:
+Researched answer: Implicit returns are returning values without needing to use the word return. If a return is the last expression then there's no need to return again.
 
 4. What is a block in Ruby?
 
-Your answer:
+Your answer: A block in ruby is usually a method that doesn't belong to an object. Blocks usually look like actual blocks of code. 
 
-Researched answer:
+Researched answer: A block in ruby is basically a method that doesn't belong to an object. These blocks are useful because it lets you save code and use it later. 
 
 1. How do you extract a value in a Ruby hash?
 
-Your answer:
+Your answer: I know that hashes are used in Ruby and I would love to send you more infomation once I research this topic further. 
 
-Researched answer:
+Researched answer: In order to extract a hash in Ruby, you use this ( =>) followed by the key name in brackets.
```

Comment:
Your researched answers were good, although at times could be expanded a little further.  For your initial answers, I would recommend studying some of the vocab to really strengthen these concepts.